### DISCUSSION TITLE
API for Mi Body Composition Scale

### DISCUSSION THREAD

**Discussion by Javison** [zepp-health/zeppos-forum#148] [Q&A] [answered]
Hi! I have a scale "Mi Body Composition Scale 2" and use Zepp Life app to connect. Does this data have an API / endpoint or is planned? (time, weight, bodyWaterRate, boneMass, muscleRate, visceralFat..)

Let me know if this is not the appropriate forum and I should post elsewhere.

Thanks in advance!

**Comment by MyoungXUE** [comment, marked as answer]
Hello, Zepp Health does not officially provide an API for the scale.
